Firestore/core/src/firebase/firestore/remote/grpc_unary_call.cc

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
If this file is meant to run on all platforms, don't forget to add it to the cmakelists.txt file.

Reply:
I'd also prefer to defer this one to a follow-up focusing on CMake.